### PR TITLE
Hazard Analysis - Address feedback from TA meeting on my parts

### PR DESCRIPTION
**Pull request by kyle4908**

### CODE REVIEW

File: docs/HazardAnalysis/HazardAnalysis.tex [hunk 1]
```diff
@@ -125,34 +125,40 @@ \section{Failure Mode and Effect Analysis}
...
       Signal Blocked \newline b. Use of inaccurate Geolocation API
       \newline c. Out of date third-party databases \newline d.
-      Device does not have a GPS \newline e. VPN usage & Error
-      messages and logging & a. Report inaccurate location detection
-      to manager & SR-AR1 & H1-1 \\
+      Device does not have a GPS \newline e. VPN usage & System
+      provides error message to user when they get geo-blocked and
+      sends an email to manager indicating that the user has been
+      geo-blocked & a. User reports inaccurate location detection to
+      manager & SR-AR1 & H1-1 \\
```
Comment (Spitgranger) [inline docs/HazardAnalysis/HazardAnalysis.tex:132]: Just interested these Safety Requirements SR-AR1 etc have been mentioned here but aren't actually described in the next section. Are we planning to this later?

**Review by Spitgranger — APPROVED** [review]

File: docs/HazardAnalysis/HazardAnalysis.tex (hunk 1 above)
Comment (kyle4908) [reply docs/HazardAnalysis/HazardAnalysis.tex:132]: oh those are pre-existing ones that I've mentioned here. I think the next section is for new ones only

Comment (Spitgranger) [reply docs/HazardAnalysis/HazardAnalysis.tex:132]: So these have already been mentioned in the SRS?

Comment (kyle4908) [reply docs/HazardAnalysis/HazardAnalysis.tex:132]: Yeah they are

**Pull request merged**